### ISSUE TITLE
Timeout Error

### ISSUE THREAD

**Issue by tmccormack165** [EzerIT/BibleOL#15]
The timeout function for exams is sometimes malfunctioning. An example of an exam that is always working is test 5, and an exam that sometimes fails is SDATSAU Final Exam Valentin. The error is not reproducible.

**Comment by oliverglanz** [comment]
Thanks for reporting this! I just need to correct the "not reproducible". We can reproduce it with the exam that malfunctions. Thus, if an exam malfunctions it always malfunctions. If an exam does not malfunction it never malfunctions. We just do not know at this moment what it is that makes an exam malfunction.

**Comment by oliverglanz** [comment]
This has been solved and the problem has been identified. This is the message I wrote to Timothy and Adrian:

I was able to identify the problem! It has to do with the amount of questions one sets. If you use an exercise as an exam, you cannot set a limit to the questions in the exercise. The number needs to stay "0":

<img width="604" alt="image1" src="https://github.com/EzerIT/BibleOL/assets/8409738/8dc1477f-dccd-4e76-a905-3f70bc330f50">


If you set it to a fixed number, the deadline feature of the exam mode will not function. So do not write something like this:

<img width="590" alt="image2" src="https://github.com/EzerIT/BibleOL/assets/8409738/9213f122-d276-4378-b942-a6f15e27be75">


The number of questions must be set in the exam mode:

<img width="795" alt="image3" src="https://github.com/EzerIT/BibleOL/assets/8409738/79cff30c-5da5-4ba1-930b-65b952b487e8">


Conclusion: Problem solved! Well, we need to improve the documentation and inform the user that he/she cannot set a fixed number of questions in the exercise that is being used for the exam.
